Startup.cs

```diff
@@ -36,8 +36,9 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
 
-            services.AddControllersWithViews();
-            IdentityModelEventSource.ShowPII = true;
+
+                       services.AddControllersWithViews();
+                              IdentityModelEventSource.ShowPII = true;
```

Comment:
I assume you used this to debug your code? But you shold remove it again if it is not needed in production